### PR TITLE
Redo #164

### PR DESCRIPTION
**Pull request by nickvergessen**
Redo #164

### CODE REVIEW

**Comment by nickvergessen** [comment]
@lordsimal

File: js/script.js [hunk 1]
```diff
@@ -29,53 +29,90 @@
...
 
-		var x = document.getElementById('swapbox');
-		x.style.backgroundColor = 'rgba(100, 100, 100, 0.8)';
+		var swapbox = document.getElementById('swapbox');
+		swapbox.style.backgroundColor = 'rgba(100, 100, 100, 0.8)';
 
-		resizeSystemCharts();
-		updateActiveUsersStatistics();
-		updateShareStatistics();
+		var updateTimer = setInterval(updateInfo, 300);
```
Comment (kesselb) [inline js/script.js]: Please remove this line. A few lines lester there should be a setTimeout call starting the updateInfo process.

Comment (nickvergessen) [reply js/script.js]: Please lets get this PR in and fix stuff later :see_no_evil:

**Comment by kesselb** [comment]
OK ;) This will basically revert https://github.com/nextcloud/serverinfo/pull/174 but I can redo it.

**Comment by nickvergessen** [comment]
> OK ;) This will basically revert #174 but I can redo it.

Yeah, sorry. merged now :tada: :ship: 

Feel free to redo :)

**Pull request merged**